internal/datastore/crdb/reader.go

```diff
@@ -36,6 +36,9 @@ var (
 		colUsersetRelation,
 		colCaveatContextName,
 		colCaveatContext,
+		colIntegrityKeyID,
+		colIntegrityHash,
+		colTimestamp,
```

Comment:
do we need the timestamp if we have the key id?

Reply:
Yes, to ensure that if a key is expired, we can still accept relationships signed from before the expiration time